api-guidelines/global/core-principles/documentation.md

```diff
@@ -1,57 +1,39 @@
+## Accessibility and usability
 
-**Discoverable and easy to digest**
-
-- All information is easily discoverable, easy to digest, and prepared in a way that the user can effectively work with it.
+- We provide access to all API endpoints in a central location, some sort of documentation portal at the best, to ensure that users can easily use the API.
+- The documentation is browsable.
+- A well-structured sidebar ensures that the documentation can be navigated so that all the endpoints can be accessed.
+- We give users the chance to get in touch with us.
+- All information is easily discoverable, easy to digest, and prepared in a way that users can effectively work with it.
 - We'll be consistent with our tone of voice, terminology, attribute names, API endpoint design, requests and responses, and counting.
-- The layout supports accordingly, e.g. with syntax highlighting or multi-column layout.
-  :::
+- The layout supports usability accordingly, e.g. with syntax highlighting or multi-column layout.
 
-::: accordion Code examples
-**Default values**  
-Request and response examples that belong to every endpoint documentation have meaningful default values that follow a specific story.
+## Code examples
 
-**Code examples and schemas**  
-Every endpoint documentation comes with examples and also provides schemas listing the available attributes with explanatory text.
+- Every endpoint documentation comes with examples and also provides schemas listing the available attributes with explanatory text.
+- Request and response examples that belong to every endpoint documentation have meaningful default values that follow a specific story.
+- Code examples and explanatory text are visually separated.
+- API endpoints and examples are shown in context.
+- API reference documentation is created and published automatically, see [MUST provide API specification using OpenAPI](../../rest/contract/openapi/rules/must-provide-api-specification-using-openapi-for-rest-apis.md).
 
-**Automated API reference documentation**  
-API reference documentation is created and published automatically, see [MUST provide API specification using OpenAPI](../../rest/contract/openapi/rules/must-provide-api-specification-using-openapi-for-rest-apis.md).
-:::
+## Context and walkthroughs
 
-::: accordion Consistency and accessibility
+- We give context for API usage such as information about HTTP methods, API operations, request and response headers, versioning, pagination, or errors.
+- With step-by-step walkthroughs we'll cover specific functionality that can be implemented with the API.
+- We cater for experienced users that already have an idea of the endpoints to be used, and at the same time provide the respective structure and information for inexperienced users who need a thorough introduction and/or example use cases.
```

Comment:
Sidenote: We definitely need to update https://github.com/otto-ec/ottoapi_portal/wiki/Language-&-Grammar#andor in order to allow this edge case :D "and or or" would be pretty weird in this case :D

Reply:
Navigated around it ;-)